include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -269,6 +269,9 @@ static const char* const kOrtSessionOptionEpContextEmbedMode = "ep.context_embed
 // in case user need to merge/connect multiple EPContext nodes in one model
 static const char* const kOrtSessionOptionEpContextNodeNamePrefix = "ep.context_node_name_prefix";
 
+// Share EP related resources across EPs
+static const char* const kOrtSessionOptionShareEpContexts = "ep.share_ep_contexts";
```

Comment:
I could be misremembering, but did you mention that this was going to be moved to qnn provider options?

Reply:
We decide to keep it as session option from our discussion. @jywu-msft

Reply:
yes, seems like sharing of contexts could be a general concept that EP's could opt into.